docs/user/testing-metrics.md

```diff
@@ -12,6 +12,14 @@ These functions expose a way to inspect and validate recorded metric values with
 
 <div data-lang="Kotlin" class="tab">
 
+Using the Glean SDK's unit testing API requires adding [Robolectric 4.0s or later](http://robolectric.org/) as a testing dependency. In Gradle, this can be done by declaring a `testImplementation` dependency:
```

Comment:
"Robolectric 4.0s"  Is the "s" intentional here?

Reply:
Oops.  No.